### DISCUSSION TITLE
Is the main line still being maintained?

### DISCUSSION THREAD

**Discussion by somnifex** [debauchee/barrier#1969] [General]
I noticed that it has been nearly 2 years since the last mainline update. At the same time, many PR fixed some problems, such as interconnection problems under different scaling ratios. Would you like to ask if the barrier mainline is still being maintained?

**Comment by DerDreschner** [comment]
It's considered unmaintained at the moment. The remaining developers created a fork of barrier, called [input-leap](https://github.com/input-leap/input-leap). But they don't have a stable release yet. They plan to release a v3.0.0 as first next-gen version of barrier. So, keep an eye open on that repo and their release page!
